public/class-admwswp-public.php

```diff
@@ -170,7 +170,7 @@ public static function weblinkWidget($attr)
                     'cart_url' => '',
                     'screen_size' => 'desktop',
                     'pager_type' => 'loadMore',
-                    'show_cart_buttons' => true,
+                    'show_cart_buttons' => !empty(get_field('showAddToCart', 'options')) ? get_field('showAddToCart', 'options') : false,
```

Comment:
It was defaulting to `true` before should we not do that if the global config isn't set?